packages/components/navbar/examples/BasicNavbarExample.tsx

```diff
@@ -1,49 +1,90 @@
 import React from 'react';
-import { Navbar } from '@contentful/f36-components';
-import { EntryIcon, ImageSquareIcon } from '@contentful/f36-icons';
+import { Navbar } from '@contentful/f36-navbar-alpha';
+import {
+  ImageSquareIcon,
+  PaintBrushIcon,
+  PenNibIcon,
+  PuzzlePieceIcon,
+  WrenchIcon,
+} from '@contentful/f36-icons';
 
```

Comment:
QQ: if we are moving from v4 to v5 should the icons not be imported from `alpha` or is this blocked by the icon migration?

Reply:
It's dependent on the ticket you are working on. So if we are able to finish your ticket first, i will update this branch accordingly.